### DISCUSSION TITLE
关于跨域的问题

### DISCUSSION THREAD

**Discussion by hengshizhi** [share121/inter-knot#1388] [其他]
这个可以所有域名的东西吗(坏笑)

**Comment by hengshizhi** [comment]
应该说是"这个可以访问所有域名的东西吗"

**Comment by share121** [comment]
并不是，我加了限制，只能访问到极其有限的接口。
目前，脚本只能访问 Github 的用户信息、Github Discussins
并且对访问范围进行精确控制，如获取用户信息。我只会获取用户名、头像、用户主页，其他信息根本不会获取
